src/components/modals/infoModal/styled.ts

```diff
@@ -4,4 +4,23 @@ export const InfoModal = styled.div`
   p {
     margin-bottom: 8px;
   }
+  kbd {
```

Comment:
Can't tell what `kbd` stands for, maybe rename to `keybind`? (I love how it shows up btw!)

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd